ruby_connectors/spec/services/connector_service_spec.rb

```
@@ -14,32 +14,32 @@
     }
   end
 
-  describe ".connect_and_fetch_status", :vcr do
+  describe ".connect_and_fetch_status" do
     it "calls connect_to_connector with the connector" do
-      VCR.use_cassette("connector_service_connect_and_fetch_status") do
+      VCR.use_cassette("connector_service/connector_service_connect_and_fetch_status") do
         expect(described_class).to receive(:connect_to_connector).with(connector)
```

rubocop

[convention] Prefer `have_received` for setting message expectations. Setup `described_class` as a spy using `allow` or `instance_spy`.

(RSpec/MessageSpies)